src/transition.lisp

```diff
@@ -115,8 +115,8 @@ Return two values:
 (defmethod transition ((qvm pure-state-qvm) (instr quil:measure))
   (values
    (measure-and-store qvm
-            (quil:qubit-index (quil:measurement-qubit instr))
-            (quil:measure-address instr))
+		      (quil:qubit-index (quil:measurement-qubit instr))
```

Comment:
Are tab stops being used? Please `C-x h M-x untabify`. Tabs won't work so well with Lisp, which wants absolute indentation, not relative.